sys/include/net/fib.h

```diff
@@ -38,7 +39,14 @@ typedef struct rp_address_msg_t {
     uint32_t address_flags; /**< The flags for the given address */
 } rp_address_msg_t;
 
-#define FIB_MSG_RP_SIGNAL (0x99)     /**< message type for RP notifications */
+/** message type for RP notification: unreachable destination */
```

Comment:
there is something wrong with the doc here and the following `#define`s. You could use `@name` and warp those `#define`s withtin `/** @{ */ .. /** @} */` or use `@brief` for each?